deploy/kubernetes/fluentd-sumologic.yaml.tmpl

```diff
@@ -1,59 +1,27 @@
+# This file is auto-generated. Please DO NOT modify.
```

Comment:
I'm nervous customers will get the wrong idea reading `Please DO NOT modify.`, since yaml customers should be allowed to modify after downloading.

Could we remove `Please DO NOT modify`.  We could actually have the CI fail the Travis build if it detects changes to this file?

Reply:
Sure that seems better. I forgot customers should still be able to change this file

Reply:
We probably can't have CI to block change to this file because CI will run on the automated commit that generates this file...

Reply:
You can use this one-liner to check the commit's author and grep -i for "travis":
```sh
if git --no-pager show -s --format="%an" . | grep -q -i "travis"; then
  # commit is from travis
fi
```

Reply:
example:
```sh
$ git log -n 1
commit 479b63ab5f612679d4ec9213482c970c09d1f442 (HEAD -> maisie-yaml-automation, origin/maisie-yaml-automation)
Author: Travis CI User <travis@example.org>
Date:   Fri Aug 9 22:32:19 2019 +0000

    Generate new 'fluentd-sumologic.yaml.tmpl'

$ git --no-pager show -s --format="%an" .
Travis CI User
```

Reply:
ah thanks Ryan

Reply:
Added check in build.sh to fail the build if the committed files contain `fluentd-sumologic.yaml.tmpl` _and_ the author of the last commit is not Travis (since auto-generated yaml should always be the last commit)